Dockerfile

```
@@ -32,4 +32,7 @@ COPY --from=builder /artifacts /bin
 # Ensure the latest CA certs are present to authenticate SSL connections.
 RUN apk --update add ca-certificates
 
+RUN addgroup -g 1001 -S flyte && adduser --uid 1001 -S flyte -G flyte
+USER flyte
```

I've finally found time to actually test this end to end... I found that you will have to make it look like this (keeping names and ids the same for convenience):

```suggestion
RUN addgroup -g 65534 -S 65534 && adduser --uid 1001 -S 1001 -G 65534
USER 1001
```

My working deployment: https://github.com/flyteorg/flyte/pull/1798/files